### ISSUE TITLE
User preferences are not editable when user session expired

### ISSUE THREAD

**Issue by musienko-maxim** [eclipse-che/che#19111]
### Che version

 - [x ] latest
 - [ ] nightly
 - [ ] other: please specify

### Steps to reproduce
Add a private registry: User preferences -> Add Container Registry ->fill the fields
Weit while session expired
Try to edit registry again
Expected result: 

### Expected behavior
the editing of  preferences should be available
Actual result: http 401 error.
### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [x] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Screenshots
![Selection_019](https://user-images.githubusercontent.com/1640058/108513578-76ae6400-72cb-11eb-9b5c-d91b4108f11b.png)
-----
![screencast-che-eclipse-che apps ci-ln-scpk94t-f76d1 origin-ci-int-gce dev openshift com-2021 02 19-14_38_48](https://user-images.githubusercontent.com/1640058/108515159-6eefbf00-72cd-11eb-8e95-743b2263efd5.gif)

### Installation method
   - [x] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [x] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by sleshchenko** [comment]
I assume here we had a valid session, but dashboard used the expired token and it was fixed in https://github.com/eclipse/che-dashboard/pull/190.
The last time I check - user is redirected to login page, when session is expired on the first operation. Rechecking and if it's true - will close the issue

**Comment by sleshchenko** [comment]
I waited until my refresh token is expired and the first time token was recovered. I don't understand how, on the screenshot see one request failed with 400 another was successful (see screenshot 1), maybe I waited not enough.
The next try, I saw login page after I returned back to IDE tab in 9 minutes. See screenshots 1 and 2.

1:
![Screenshot_20210325_163323](https://user-images.githubusercontent.com/5887312/112490223-dc3cb700-8d87-11eb-90b8-bf948e79dc4a.png)

2:
![Screenshot_20210325_163259](https://user-images.githubusercontent.com/5887312/112490560-23c34300-8d88-11eb-872e-de454f409a57.png)

I believe it's fully solved by https://github.com/eclipse/che-dashboard/pull/190, if you face it again, create a new one or reopen this.
